### PR TITLE
July data entry Fixes

### PR DESCRIPTION
**Pull request by InfamousSky**

### CODE REVIEW

File: packs/data/blood-lords-bestiary.db/pesgahi-the-poisoner.json [hunk 1]
```diff
@@ -992,15 +992,17 @@
...
                     },
                     {
                         "key": "Note",
                         "outcome": [
                             "criticalSuccess"
                         ],
                         "selector": "{item|_id}-damage",
-                        "text": "PF2E.BombNotes.AcidFlask.Moderate.criticalSuccess"
+                        "text": "PF2E.BombNotes.AcidFlask.Moderate.criticalSuccess",
+                        "title": ""
```
Comment (SpartanCPA) [inline packs/data/blood-lords-bestiary.db/pesgahi-the-poisoner.json:1005]: Why a blank title?

Comment (InfamousSky) [reply packs/data/blood-lords-bestiary.db/pesgahi-the-poisoner.json:1005]: i don't know what that is so it might have been auto added somehow?

Comment (TikaelSol) [reply packs/data/blood-lords-bestiary.db/pesgahi-the-poisoner.json:1005]: Make it `{item|name}`

Comment (InfamousSky) [reply packs/data/blood-lords-bestiary.db/pesgahi-the-poisoner.json]: added the `{item|name}`s to the titles

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**